### PR TITLE
gh-issue: add page

### PR DESCRIPTION
**Pull request by TheLastGimbus**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Add a new page for `gh issue`

Ref: #4701

### CODE REVIEW

File: pages/common/gh-issue.md
Comment (navarroaxel) [inline pages/common/gh-issue.md:10]: I prefer to get emphasis to what the command does, and not what the user does in the browser itself.
```suggestion
- Open a browser window for create a new issue:
```

File: pages/common/gh-issue.md
Comment (navarroaxel) [inline pages/common/gh-issue.md:16]: We prefer long argument format:
```suggestion
`gh issue list --limit {{count}} --label {{label}}`
```

File: pages/common/gh-issue.md
Comment (navarroaxel) [inline pages/common/gh-issue.md:14]:
```suggestion
- List issues with a specific label:
```

File: pages/common/gh-issue.md
Comment (navarroaxel) [inline pages/common/gh-issue.md:20]: In settings is called `username`:
```suggestion
`gh issue list --state closed --author {{username}}`
```

File: pages/common/gh-issue.md
Comment (navarroaxel) [inline pages/common/gh-issue.md:18]: I think `authored` is the correct term
```suggestion
- List closed issues authored by a specific user:
```

File: pages/common/gh-issue.md
Comment (navarroaxel) [inline pages/common/gh-issue.md:24]:
```suggestion
`gh issue status --repo {{owner}}/{{repository}}`
```

File: pages/common/gh-issue.md
Comment (navarroaxel) [inline pages/common/gh-issue.md:22]: I think `display` is better:
```suggestion
- Display status of issues relevant to you, in a specific repository:
```

File: pages/common/gh-issue.md
Comment (navarroaxel) [inline pages/common/gh-issue.md:1]:
```suggestion
# gh issue
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @TheLastGimbus, I left some comments.

**Comment by TheLastGimbus** [comment]
I commited some of changes, but for the rest (and also applying for my 2 other `gh command` PRs), I have a question:

I'm new here, and I'm obiously not a maintainer, but from what I see (or would want) TL;DR pages to be - a quick, beginner friendly/human-readable examples for how to do common tasks

That's why I 
 - used more human'ish terms - like "made" instead of "authored"
 - focused on "what the user does in the browser itself", that is, what the _user_ wanted in the first place, not what the _command does_.

That's my point of view - if this repo has strict design ways and you want me to change them to be more technical - it's all right :+1:

File: pages/common/gh-issue.md
Comment (TheLastGimbus) [reply pages/common/gh-issue.md:16]: In here (reffering to what I wrote in a comment) - I wanted to focus on practical example "what the user might want to do", instead of boring listing what are available options - that's what `man pages` and `--help` already do :confused:

File: pages/common/gh-issue.md
Comment (navarroaxel) [reply pages/common/gh-issue.md:14]:
```suggestion
- List 10 issues with the `bug` label:
```

File: pages/common/gh-issue.md
Comment (navarroaxel) [reply pages/common/gh-issue.md:16]:
```suggestion
`gh issue list --limit {{10}} --label "{{bug}}"`
```

File: pages/common/gh-issue.md
Comment (navarroaxel) [reply pages/common/gh-issue.md:18]:
```suggestion
- List closed issues made by a specific user:
```

**Comment by navarroaxel** [comment]
@TheLastGimbus Don't worry, just apply a few style changes that I suggested and we can check if the page is correct or need more work.

File: pages/common/gh-issue.md
Comment (TheLastGimbus) [reply pages/common/gh-issue.md:10]: Here, again, I wanted to focus more on "what user wants to do", not what technically command does

Comment (sbrl) [reply pages/common/gh-issue.md:10]: Hrm, I see your point @TheLastGimbus. I don't think it really matters either way here - I think I actually prefer @TheLastGimbus' original example. They both have merit though, since @navarroaxel's highlights the fact that a new web browser window is opened to create the issue, rather than the browser window being opened in

File: pages/common/gh-issue.md
Comment (sbrl) [inline pages/common/gh-issue.md:22]: We do have a convention surrounding the use of the 2nd person though (e.g. `you`, `your`), in that we avoid it if at all possible. What about this?
```suggestion
- Display the status of relevant issues in a specific repository:
```

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the new page! I've left a comment below for you to review.

File: pages/common/gh-issue.md
Comment (TheLastGimbus) [reply pages/common/gh-issue.md:22]: Umm, this command specifically shows *your* issues, so I would change that to at least "relevant to user" - is that ok?

Comment (owenvoke) [reply pages/common/gh-issue.md:22]: Makes sense to me. 👍🏻

**Review by owenvoke — APPROVED** [review]
LGTM after the requested change has been resolved. 👍🏻

**Review by navarroaxel — APPROVED** [review]

**Comment by owenvoke** [comment]
@sbrl, is this ok to merge? Looks like your change has been applied. 👍🏻

**Review by sbrl — APPROVED** [review]
Oh, thanks for reminding me @owenvoke!

Looks ok to me :D

**Pull request merged**
